portality/bll/services/site.py

```diff
@@ -1,13 +1,57 @@
-from portality.core import app
-from portality import models
-from portality.store import StoreFactory, prune_container
+import re
 from datetime import datetime
-from portality.lib.argvalidate import argvalidate
+from typing import Iterable
+
+import werkzeug.routing
+from lxml import etree
 
+from portality import models
 from portality.bll import exceptions
+from portality.core import app
+from portality.lib.argvalidate import argvalidate
+from portality.store import StoreFactory, prune_container
+from portality.util import get_full_url_by_endpoint
+
+NS = "{http://www.sitemaps.org/schemas/sitemap/0.9}"
+
+
+def create_simple_sub_element(parent, element_name, text=None):
```

Comment:
Propose to move this function to `portality.xmlutil`

Reply:
unfortunately, this function contain logic for sitemap.

let's extract or refactor in future if other module need function similar to `create_simple_sub_element`